_posts/2021-02-04-pulpcore-3.10-is-now-generally-available.md

```diff
@@ -0,0 +1,143 @@
+
+
+## New filter capability for user and group endpoints
+
+You can now filter lists of users and groups.  [#7975](https://pulp.plan.io/issues/7975)
+
+The users endpoint now has the following filter options:         
+
+```
+fields = {
+     "username": ["exact", "iexact", "in", "contains", "icontains"],
+     "first_name": ["exact", "iexact", "in", "contains", "icontains"],
+     "last_name": ["exact", "iexact", "in", "contains", "icontains"],
+     "email": ["exact", "iexact", "in", "contains", "icontains"],
+     "is_active": ["exact"],
+     "is_staff": ["exact"],
+ }
+
+```
+
+The group endpoint now has the following filter options:
+
+```
+fields = {
+         "id": ["exact", "in"],
+         "name": ["exact", "iexact", "in", "contains", "icontains"],
+     }
+```
+
+
+## Add your own labels to objects
+
+With the addition of the `pulp_labels` field, you can create labels to add your own key/data for your resources in Pulp. You can also modify and remove these labels if needed.  
+
+The `pulp_label_select` filter was added so that you can now filter by the labels you assign.
+
+The `Label` and `LabelSerializer` have also been added to the Plugin API. [#8065](https://pulp.plan.io/issues/8065)
+
+
+For more information, see the [Labels](https://docs.pulpproject.org/pulpcore/workflows/labels.html) documentation.
+
+## New optional Header field added for aiohttp ClientSession
+
+You now have the option to specify headers in aiohttp ClientSession. Headers must be
+provided as a dictionary or a JSON field. You can set headers for an entire
+client session or per request. [#8083](https://pulp.plan.io/issues/8083)
+
+## RBAC
+
+There have been a number of features added that relate to the ongoing effort to add role based access control (RBAC) functionality to Pulp.
+
+* Users with view group permissions can view groups. [#8159](https://pulp.plan.io/issues/8159)
+* Users with change group permissions can modify groups. [8160](https://pulp.plan.io/issues/8160)
+* Authenticated users can create and view artifacts. [#8193](https://pulp.plan.io/issues/8193)
+* Filtering has been added for access policies.  [#8189](https://pulp.plan.io/issues/8189)
+
+
+### New AccessPolicy.customized field.
+
+The `AccessPolicy.customized` field is a read-only field that defaults to `False`.
+If `True`, this indicates that the user has modified the default access policy. [#8182](https://pulp.plan.io/issues/8182)
+
+This will be useful for:
+
+* Users to see how a system's RBAC has been modified
+* Plugin writers who want to ship changes or bugfixes to default policies they previously shipped and don't want to overwrite a user's custom policy.
+
+## Migration imports now use the `apps.get_model`
+In various places in the migrations, models are imported directly and then used. This was a hazard because when the model definition changes, this code can break.
+
+The data migrations now use `apps.get_model(...)` instead. [#8203](https://pulp.plan.io/issues/8203)
```

Comment:
Not sure if it's worth calling this out. It's not a user facing item and not plugin API fix/feature as well.

Reply:
I will remove then!